bdai_ros2_wrappers/bdai_ros2_wrappers/action_handle.py

```diff
@@ -72,8 +75,7 @@ def wait_for_result(self, timeout_sec: Optional[float] = None) -> bool:
         )
 
     def wait_for_acceptance(self, timeout_sec: Optional[float] = None) -> bool:
-        """
-        Waits until goal is accepted or timeout.
+        """Waits until goal is accepted or timeout.
```

Comment:
@amessing-bdai nit: do we have to make this change? It worsens readability IMO.

Reply:
I think this was auto-done by ruff (as were most of the formatting documentation changes). We can remove which rule caused this, but I didn't pick that list so we will be  different from bdai.

Reply:
Looked through the ruff rules. Our style guide follows google's and they have the rule that causes this: https://docs.astral.sh/ruff/rules/multi-line-summary-first-line/

Reply:
It's OK. Consistency is good.